akka-javasdk-testkit/src/main/java/akka/javasdk/testkit/EventSourcedTestKit.java

```diff
@@ -41,7 +52,29 @@ private EventSourcedTestKit(ES entity, String entityId) {
    */
   public static <S, E, ES extends EventSourcedEntity<S, E>> EventSourcedTestKit<S, E, ES> of(
       Supplier<ES> entityFactory) {
-    return of("testkit-entity-id", entityFactory);
+    return of(DEFAULT_TEST_ENTITY_ID, entityFactory);
+  }
+
+  /**
+   * Creates a new testkit instance from a Supplier of EventSourcedEntity and a state into which
+   * the supplied entity will be placed for tests.
```

Comment:
Is this sentence correct? A state into which the supplied entity will be placed? 

The supplied entity is placed into the testkit, but into the state. 
But I think that we can make it simpler without exposing users to too much details.

```suggestion
   * Creates a new testkit instance from a Supplier of EventSourcedEntity and a state.
```

So, the testkit is created using supplier + state. 
We don't need to explain that the testkit instance is wrapping something. I guess users can infer it by using it.